elasticsearch/run.sh

```diff
@@ -35,11 +38,18 @@ BYTES_PER_GIG=$((1024*${BYTES_PER_MEG}))
 MAX_ES_MEMORY_BYTES=$((64*${BYTES_PER_GIG}))
 MIN_ES_MEMORY_BYTES=$((256*${BYTES_PER_MEG}))
 
+if [[ -e $MEM_LIMIT_FILE ]]; then
```

Comment:
Do we need to check the existence of the file?

Reply:
yes, if the file doesn't exist because the DC doesn't have the volume mount defined then we don't want to fail